### PR TITLE
TypeScript 4.1

### PR DESCRIPTION
**Pull request by alan-agius4**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,30 +42,30 @@
...
+    "@angular-devkit/build-angular": "^0.1101.0-next.0",
+    "@angular-devkit/core": "^11.1.0-next.0",
+    "@angular-devkit/schematics": "^11.1.0-next.0",
+    "@angular/animations": "^11.1.0-next.0",
+    "@angular/bazel": "^11.1.0-next.0",
+    "@angular/cli": "^11.1.0-next.0",
+    "@angular/common": "^11.1.0-next.0",
+    "@angular/compiler": "^11.1.0-next.0",
+    "@angular/compiler-cli": "^11.1.0-next.0",
+    "@angular/core": "^11.1.0-next.0",
```
Comment (kyliau) [inline package.json:55]: latest should be `-next.1`

File: tools/defaults.bzl [hunk 2]
```diff
@@ -31,13 +31,13 @@ def ts_library(tsconfig = None, deps = [], testonly = False, **kwargs):
...
-NG_VERSION = "^11.0.0"
+NG_VERSION = "^11.1.0-next.0"
 RXJS_VERSION = "^6.5.5"
 HAPI_VERSION = "^18.4.0"
 EXPRESS_VERSION = "^4.15.2"
 EXPRESS_TYPES_VERSION = "^4.17.0"
-DEVKIT_CORE_VERSION = "^11.0.0"
-DEVKIT_ARCHITECT_VERSION = "^0.1100.0"
+DEVKIT_CORE_VERSION = "^11.1.0-next.0"
+DEVKIT_ARCHITECT_VERSION = "^0.1101.0-next.1"
```
Comment (kyliau) [inline tools/defaults.bzl:40]: How come this version is different from the one above?

**Review by kyliau — APPROVED** [review]
LGTM after cleanup

File: tools/defaults.bzl (hunk 2 above)
Comment (alan-agius4) [reply tools/defaults.bzl:40]: Because this is an experimental package.

Comment (kyliau) [reply tools/defaults.bzl:40]: I meant the patch version. DEVKIT_CORE is `-next.0` whereas DEVKIT_ARCHITECT is `-.next.1`

Comment (alan-agius4) [reply tools/defaults.bzl:40]: Oh right!

**Pull request merged**